Parkinson/Startup/APHProfileExtender.m

```diff
@@ -33,7 +33,17 @@
  
 #import "APHProfileExtender.h"
 
-static  NSInteger  kDefaultNumberOfExtraSections =  2;
+// ***************************************************************************
+// ***************************************************************************
+// Of Note: This class is adding extra sections as of mPower 1.1 because no special
+// options were still required. In order to add special sections, set the
+// kDefaultNumberOfExtraSections to the # required and adjust the decorateCell
+// and didSelectorRowAtIndexPath delegate methods to reflect the new section(s)
+// desired.
+// ***************************************************************************
+// ***************************************************************************
+
+static  NSInteger  kDefaultNumberOfExtraSections =  0;
```

Comment:
This seems weird--it was 2, but there was only 1 extra section, wasn't there? But 2 worked for adding one section, and putting it to 0 just removes the one section? Any idea what's up with that?